config/field.storage.node.field_staff_department.yml

```diff
@@ -1,17 +1,24 @@
-uuid: 7e3831b5-8101-4587-8998-0c045b7680cb
+uuid: cd72f133-1864-4bc7-963b-83aedb6305dd
 langcode: en
 status: true
 dependencies:
   module:
     - node
-    - text
+    - options
 id: node.field_staff_department
 field_name: field_staff_department
 entity_type: node
-type: text
+type: list_string
 settings:
-  max_length: 255
-module: text
+  allowed_values:
+    -
+      value: 'Long Range Planning Staff'
+      label: 'Long Range Planning Staff'
+    -
+      value: 'The Planning Commission'
+      label: 'The Planning Commission'
+  allowed_values_function: ''
+module: options
 locked: false
 cardinality: 1
 translatable: true
```

Comment:
@JTHall94 I have a hard time letting this one slide. If, for whatever reason, this needs to expand to more than just those two things, the database / field settings can't be changed if there is data for this field in the database. List values should be used more for fixed lists that can and will never change. An example of this could be colors of the rainbow. It will always be, forever, red, orange, yellow, green, blue, purple. Just have found out the hard way that a list field is very limited. My suggestion would be to make a staff department taxonomy and make this field an entity reference to that field.